meteor-app/imports/api/models/queries/user.ts

```diff
@@ -1,56 +1,36 @@
 import sql from "imports/lib/sql-template";
-import { simpleQuery1 } from "imports/api/connectors/postgresql";
-import { withMongoDB } from "imports/api/connectors/mongodb";
+import { simpleQuery1, simpleQuery } from "imports/api/connectors/postgresql";
```

Comment:
It is not obvious what the difference between these two queries are. What would be a better name to differentiate them? It looks like simpleQuery1 is nullable so the name can query name can be changed to indicate that instead of using a "1".

Reply:
I would also add a comment stating when you should use one query vs the other

Reply:
I'll change that in a latter branch.